packages/@react-spectrum/table/src/table.css

```diff
@@ -19,17 +19,26 @@
 .react-spectrum-Table-cell {
   display: flex;
   align-items: center;
+  /* To ensure the flex child only takes up available width, see https://makandracards.com/makandra/66994-css-flex-and-min-width */
+  min-width: 0px;
```

Comment:
I don't think the min-width is needed on this one, it's only needed for a child of a flex with a shrink rule of > 0

Reply:
Isn't that the case here? The child of `react-spectrum-Table-cell` is `spectrum-Table-cellContent` which has `flex: 1 1 0%`

Reply:
I mean it's needed on the child, not on the display flex

Reply:
Ah derp, will change